CHANGELOG.md

```diff
@@ -4,6 +4,27 @@
 
 💥 Breaking changes:
 
+- Update the font to v2
```

Comment:
What do we think about splitting this out into multiple bullet points? (e.g. the IE8 change could be it's own thing) and possibly trying to give a TL;DR of how v2 is different and/or why we're doing this?